examples/outlet-form-rerender/app/routes/users.tsx

```diff
@@ -0,0 +1,34 @@
+import type { LoaderFunction, MetaFunction } from "remix";
+import { Link, Outlet } from "remix";
+import { useLoaderData } from "remix";
+import type { User } from "~/data.server";
+import { users } from "~/data.server";
+
+interface LoaderData {
+  users: User[];
+}
+
+export const meta: MetaFunction = () => {
+  return { title: "Users" };
+};
+
+export const loader: LoaderFunction = async () => {
+  return { users } as LoaderData;
```

Comment:
```suggestion
  return json<LoaderData>({ users });
```

And we'll need to import json above